### ISSUE TITLE
Add option for custom activation (e.g. activation with hyperparams).

### ISSUE THREAD

**Issue by david-thrower** [david-thrower/cerebros-core-algorithm-alpha#15]

**Comment by david-thrower** [comment]
Genomic research often will Z scale target phenotypes when they are doing regression based GWAS experiments. This makes it easy to quantify the RMSE / MAE in terms of standard deviations from the mean. This gives a metric / loss that is relative to the distribution of the data as well.

If we can extend this (Likely split off as a separate issue), to train and predict with the following strategy:

Z-scale the target -> train  and compute loss on z-scaled targets -> then produce validation metrics by producing z-scaled predictions, then returning reverse - z - scaled predictions, this may be more robust. Same with 1-0 scaling. Given the volume of data Cerebros was designed for, the slight increase in accuracy from z-scaling may be worth the marginal computational expense over 1-0 scaling.    

What is proposed is to add a parameter that allows a hyperparameter to be passed to the output Dense layer's activation and also allows both string names and functions / lambdas to be passed for the final_activation argument. with this, we may need a way to parse the function from a string, as it is good to make the final activation tunable and tuners like Katib, optuna, ... typically can't take a list of functions as the options for parameters. Basically we would need (conditionally executed when final_activation is not a string in the list of parameter-less Keras activations):

```python3
warn("If you will be training in a production system where users have access to select the argument for final_activation, do not allow the user to manually enter arguments for final_activation, make the user select from a menu of activation that passes a primary key to the back - end of the system, and have the system parse the argument on the back-end. When a string that is not a name of a Keras activation having no additional parameter is selected, the user may inject arbitrary code if they have direct access. see ")


final_activation = eval(f"{final_activation}({tuning_constant})" )

```
